redact_sample.py

```diff
@@ -0,0 +1,117 @@
+    sample_row['HTC_WILLINGNESS'] = SampleGen.get_random_htc()
+    sample_row['HTC_DIGITAL'] = SampleGen.get_random_htc()
+    if sample_row['ESTAB_TYPE'] in ['MILITARY US SFA', 'MILITARY SFA', 'MILITARY US SLA', 'ROYAL HOUSEHOLD',
+                                    'MOD HOUSEHOLDS', 'HIGH SECURE MENTAL HEALTH', 'ROUGH SLEEPER', 'TRANSIENT PERSONS',
+                                    'TRAVELLING PERSONS', 'GRT SITE', 'MIGRANT WORKERS', 'IMMIGRATION REMOVAL CENTRE',
+                                    'SHELTERED ACCOMMODATION', 'APPROVED PREMISES', 'RESIDENTIAL CHILDRENS HOME',
+                                    'RELIGIOUS COMMUNITY', 'LOW/MEDIUM SECURE MENTAL HEALTH']:
+        sample_row['ESTAB_TYPE'] = SampleGen.get_random_estab_type()
+        sample_row['ADDRESS_LINE1'] = SampleGen.get_random_address_line()
+        sample_row['ADDRESS_LINE2'] = ''
+        sample_row['ADDRESS_LINE3'] = ''
+        sample_row['TOWN_NAME'] = SampleGen.get_random_post_town()
+        sample_row['POSTCODE'] = SampleGen.get_random_post_code()
+        sample_row['LATITUDE'] = SampleGen.get_random_lat_or_long()
+        sample_row['LONGITUDE'] = SampleGen.get_random_lat_or_long()
+
+    return sample_row
+
+
+def _write_row(writer, sample_row):
+    writer.writerow({
+        'UPRN': sample_row['UPRN'],
+        'ESTAB_UPRN': sample_row['ESTAB_UPRN'],
+        'ADDRESS_TYPE': sample_row['ADDRESS_TYPE'],
+        'ESTAB_TYPE': sample_row['ESTAB_TYPE'],
+        'ADDRESS_LEVEL': sample_row['ADDRESS_LEVEL'],
+        'ABP_CODE': sample_row['ABP_CODE'],
+        'ORGANISATION_NAME': sample_row['ORGANISATION_NAME'],
+        'ADDRESS_LINE1': sample_row['ADDRESS_LINE1'],
+        'ADDRESS_LINE2': sample_row['ADDRESS_LINE2'],
+        'ADDRESS_LINE3': sample_row['ADDRESS_LINE3'],
+        'TOWN_NAME': sample_row['TOWN_NAME'],
+        'POSTCODE': sample_row['POSTCODE'],
+        'LATITUDE': sample_row['LATITUDE'],
+        'LONGITUDE': sample_row['LONGITUDE'],
+        'OA': sample_row['OA'],
+        'LSOA': sample_row['LSOA'],
+        'MSOA': sample_row['MSOA'],
+        'LAD': sample_row['LAD'],
+        'REGION': sample_row['REGION'],
+        'HTC_WILLINGNESS': sample_row['HTC_WILLINGNESS'],
+        'HTC_DIGITAL': sample_row['HTC_DIGITAL'],
+        'TREATMENT_CODE': sample_row['TREATMENT_CODE'],
+        'FIELDCOORDINATOR_ID': sample_row['FIELDCOORDINATOR_ID'],
+        'FIELDOFFICER_ID': sample_row['FIELDOFFICER_ID'],
+        'CE_EXPECTED_CAPACITY': sample_row['CE_EXPECTED_CAPACITY'],
+        'CE_SECURE': sample_row['CE_SECURE'],
+        'PRINT_BATCH': sample_row['PRINT_BATCH']})
+
+
+def main():
+    log_level = os.getenv('LOG_LEVEL')
+    logging.basicConfig(handlers=[logging.StreamHandler(sys.stdout)], level=log_level or logging.ERROR)
+    logger.setLevel(log_level or logging.INFO)
+    args = parse_arguments()
+    output_file_path = 'sample_files/' + args.sample_file_path.split('/')[-1].split('.csv')[0] + '_redacted.csv'
```

Comment:
This feels like it should have a comment explaining what it's doing and why, or be in its own well-named function so that the code is self-documenting